README.md

```diff
@@ -31,7 +31,7 @@ Samples
 
 * **[WearOAuth](WearOAuth)** (Kotlin) - Demonstrates how developers can authenticate a user on their Wear OS app via the user's mobile/phone device without requiring a mobile app (Wear OS companion app handles the request on the mobile side). The sample uses OAuth. [Guide](https://developer.android.com/training/wearables/apps/auth-wear)
 
-* **[WearSpeakerSample](WearSpeakerSample)** (Kotlin) - Demonstrates audio recording and playback if the wearable device has a speaker. [Guide](https://developer.android.com/training/wearables/wearable-sounds)
+* **[WearSpeakerSample](WearSpeakerSample)** (Compose/Kotlin) - Demonstrates audio recording and playback if the wearable device has a speaker. [Guide](https://developer.android.com/training/wearables/wearable-sounds)
```

Comment:
I would call out this is an advanced sample that uses effects and ConstraintLayout and link both of those to the guides. That way they know before even getting to the main README what it covers.

Reply:
Done!